core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -1247,7 +1300,7 @@ object GroupMetadataManager {
           member.assignment = Utils.toArray(memberMetadata.get(ASSIGNMENT_KEY).asInstanceOf[ByteBuffer])
           member
         }
-        GroupMetadata.loadGroup(groupId, initialState, generationId, protocolType, protocol, leaderId, members)
+        GroupMetadata.loadGroup(groupId, initialState, generationId, protocolType, protocol, leaderId, currentStateTimestamp, members)
       } else {
         throw new IllegalStateException("Unknown group metadata message version")
```

Comment:
Can we add the version to this message? Same for the similar case above loading offsets.